frontend/components/panels/StakePanel.tsx

```diff
@@ -57,6 +59,25 @@ export function StakePanel({ mainStakeAccount }: StakePanelProps) {
     []
   )
 
+  const [isLlcModalOpen, setIsLlcModalOpen] = useState(false)
+
+  // This only executes if deposit action is enabled
+  const onAction = useCallback(async () => {
+    if (mainStakeAccount === 'NA') setIsLlcModalOpen(true)
+    else {
+      try {
+        const isLlcMember = await stakeConnection!.isLlcMember(
```

Comment:
Man the `!` kill me everytime. Not sure how to fix them though.

Reply:
yeah, i do think there might be a solution here. will think about it.